Taarafo.Portal.Web/Brokers/Apis/IApiBroker.Groups.cs

```diff
@@ -12,5 +13,7 @@ namespace Taarafo.Portal.Web.Brokers.Apis
     public partial interface IApiBroker
     {
         ValueTask<List<Group>> GetAllGroupsAsync();
+
```

Comment:
Remove the extra line

Reply:
I have removed the extra line in the CODE RUB: Removed Extra Line commit.